<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMonsterMash</class>
 <widget class="QMainWindow" name="QMonsterMash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMonsterMash</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="60">
      <item>
       <widget class="KPlotWidget" name="kpPV"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actPump"/>
    <addaction name="actMash"/>
    <addaction name="actMashSchedule"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actBoilTimer"/>
    <addaction name="actHydrometerCorrection"/>
    <addaction name="actPlotStepResponse"/>
    <addaction name="actDilCalc"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actRegSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actVersion"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actMash"/>
   <addaction name="actPump"/>
  </widget>
  <action name="actBoilTimer">
   <property name="text">
    <string>Boil timer</string>
   </property>
  </action>
  <action name="actMashSchedule">
   <property name="text">
    <string>Mash schedule</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actHydrometerCorrection">
   <property name="text">
    <string>Hydrometer correction</string>
   </property>
  </action>
  <action name="actRegSettings">
   <property name="text">
    <string>Regulator settings</string>
   </property>
  </action>
  <action name="actPlotStepResponse">
   <property name="text">
    <string>Plot step response</string>
   </property>
  </action>
  <action name="actMash">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/startMashIcon</normaloff>:/images/startMashIcon</iconset>
   </property>
   <property name="text">
    <string>Start Mash</string>
   </property>
  </action>
  <action name="actPump">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/startPumpIcon</normaloff>:/images/startPumpIcon</iconset>
   </property>
   <property name="text">
    <string>Start pump</string>
   </property>
  </action>
  <action name="actVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actDilCalc">
   <property name="text">
    <string>Dilution calculator</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KPlotWidget</class>
   <extends>QFrame</extends>
   <header>kplotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
